plexapi/audio.py

```diff
@@ -2,6 +2,7 @@
 import os
 from pathlib import Path
 from urllib.parse import quote_plus
+from xml.etree.ElementTree import Element
```

Comment:
perhaps add such imports that only used for typing under TYPE_CHECKING check?

example:
- https://github.com/Taxel/PlexTraktSync/blob/2f1235e127d69491f3d748d54eccdec9504aef62/plextraktsync/queue/Queue.py#L5-L8

this gives back small performance loss when importing more modules

Reply:
In this case, this should have no impact on performance because `Element` is already imported in a different module.